README.md

```diff
@@ -32,6 +32,27 @@ with a Value Change Dump (VCD) or Extended VCD (EVCD) file to produce an svg wav
 - `-s | --start FORMULA` Specify the start of the window.
 - `-w | --wires LIST` Comma-separated list of wires to include in the visualization (default to all wires).
 
+## Sootty Features
+- **VCD file support:** Sootty can read and display Value Change Dump (VCD) files, a common format for recording waveform data in digital circuit design and simulation.
+
+- **Selective signal display:** Users can choose to display all signals in a waveform or only specific signals of interest, reducing clutter and focusing on relevant data.
+
+- **Zooming and panning:** Sootty allows users to zoom in on a specific section of the waveform for more detailed analysis and to pan across the waveform to explore it.
```

Comment:
Panning and zooming aren't really features yet, but it could be cool to add them! (You can zoom, but only by setting the bounds of the viewing window in the initial command.)